examples/chip-tool/commands/discover/DiscoverOperationalsCommand.h

```diff
@@ -0,0 +1,62 @@
+/*
+ *   Copyright (c) 2023 Project CHIP Authors
+ *   All rights reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ *
+ */
+
+#pragma once
+
+#include "../common/CHIPCommand.h"
+
+class DiscoverOperationalsStartCommand : public CHIPCommand
+{
+public:
+    DiscoverOperationalsStartCommand(CredentialIssuerCommands * credIssuerCommands) :
+        CHIPCommand("start-operationals", credIssuerCommands)
```

Comment:
All the commands here, and the commissionables commands, desperately need help text.  And maybe some renaming for the commissionables commands...

Given the way start/stop work, should we in fact just have a single stop command and single list command and just two different start commands, since right now we can have at most one scanner going on?